src/main/java/org/sagebionetworks/web/client/widget/user/UserBadgeViewImpl.java

```diff
@@ -187,6 +187,13 @@ public void setHref(String href) {
 	public void clearHref() {
 		usernameLink.setHref(HasHref.EMPTY_HREF);
 	}
+	
+	@Override
+	public void openNewWindow(String url) {
+		usernameLink.setTarget("_blank");
+		DisplayUtils.newWindow(url, "", "");
```

Comment:
Really interesting that the anchor target is used after the click event has taken place!  Well, I don't trust that every browser is going to handle this properly, please set the name value in the js Window open() method.
DisplayUtils.newWindow(url, "_blank", "");